.eslintrc

```diff
@@ -216,12 +210,23 @@
     },
     {
       "files": ["*.pw.ts*", "*.test-pw.ts*", "playwright/**/*.ts*"],
+      "plugins": ["playwright"],
       "rules": {
+        "playwright/no-commented-out-tests": "error",
+        "playwright/no-focused-test": "error",
+        "playwright/no-skipped-test": "warn",
```

Comment:
Use linter to flag a warning for skipped Playwright tests - to encourage us to avoid skipping tests without a reason, and to flag them as a problem that needs resolving eventually